components/TARGET_PSA/spm/doc/README.md

```diff
@@ -0,0 +1,23 @@
+# Mbed Secure Partition Manager (SPM)
+
+The Platform Security Architecture (PSA) firmware framework specifications contain a logic component called the Secure Partition Manager (SPM). PSA defines a Secure Processing Environment (SPE) for:
```

Comment:
Mbed secure partition manager, unless it's a brand name. Is it?

Reply:
the file will be removed as well